change/@fluentui-react-motion-ef078e0f-9988-442e-b6ac-5c7ee5f6a32c.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31787&commitId=dafc6dd5e7e516bfa23c3337bf9963783d792ea9&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.normal.chromium.png|0| Added|

</details>


  <details><summary><b>Card Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged.appearance - RTL.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Card Converged - Interactive</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Interactive.appearance interactive - Outline.click.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Outline.hover.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Outline.normal.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Subtle.click.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Subtle.hover.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Subtle.normal.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.click.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.hover.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>SwatchPicker Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.Shape.default.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>